stable/yugaware/values.yaml

```diff
@@ -65,6 +65,8 @@ yugaware:
   # enables features specific for cloud deployments
   cloud:
     enabled: false
+  volume:
```

Comment:
Do we want to move `storage` and `storageClass` into this new volume object? It will cause some pain for upgrading, but will be better organization for the long term.

Also, can we rename `volume` -> `pvc`? This is more of a minor issue, but it's a bit confusing to use the term "volume" standalone because there is another k8s resource called `PersistentVolume`.

Reply:
I thought about that but decided to keep backwards compatibility. Maybe we can file a bug with a "breaking-change" label or something? Then next time we release a major version we can move them into here?

Alternatively I can name this `storageAnnotations` which is somewhat mismatched with `service` but and `pod` but matches the other storage options.

How about `persistentVolumeClaim` so that non-kube experts can still grok what this is about without looking up every acronym in the file. (Not that "volume" is a great name to start with...) I also wouldn't mind just calling it `storage` or `disk`.

Reply:
I'm happy with just `storageAnnotations` so it matches the other storage fields. We should also move it to line 42 so that it's colocated with the other storage values.

Reply:
Done!